src/Matrix/index.js

```diff
@@ -5,120 +5,159 @@ Use: M.Matrix.functionality('string')
-		}
-		return secondary;
+					let secondary = [], p = 0;
+					for (let i = 0; i < arr.length; i++) {
+						secondary[p++] = arr[i][arr.length - i - 1];
+					}
+					return secondary;
 
-		default:
-			throw new TypeError(`Invalid argument '${type}' received!`);
+				default:
+					throw new TypeError(`Invalid argument '${type}' received!`);
 			}
 		}
 	}
 
-	check(property){
+	check(property) {
 
 		stringValidate(property, 'check');
 		let arr = this.args;
 
-		switch(property){
+		switch (property) {
 
-		case 'square':
-			return (checkSquare(arr) === 1)? true: false;
+			case 'square':
+				return (checkSquare(arr) === 1) ? true : false;
 
-		case 'identity':
-			if(checkSquare(arr)){
-			for(let i = 0, j = 0; i < arr.length && j < arr[0].length; i++, j++){
-					if(i === j && arr[i][j] !== 1 && arr[j][i] !== 0){
+			case 'identity':
+				if (checkSquare(arr)) {
+					for (let i = 0, j = 0; i < arr.length && j < arr[0].length; i++ , j++) {
+						if (i === j && arr[i][j] !== 1 && arr[j][i] !== 0) {
 							return false;
 						}
 					}
 					return true;
 				}
 				throw new TypeError(`Invalid argument received: '${arr}', the matrix has to be square to check '${property}' property!`);
 
-		case 'upper-triangular':
-			if(checkSquare(arr)){
-			for(let i = 1, j = 0; i < arr.length && j < i; i++){
-					if(arr[i][j] !== 0){
-						return false;
+			case 'upper-triangular':
+				if (checkSquare(arr)) {
+					for (let i = 1, j = 0; i < arr.length && j < i; i++) {
+						if (arr[i][j] !== 0) {
+							return false;
 						}
 					}
-				return true;
-			}
-			throw new TypeError(`Invalid argument received: '${arr}', the matrix has to be square to check '${property}' property!`);
+					return true;
+				}
+				throw new TypeError(`Invalid argument received: '${arr}', the matrix has to be square to check '${property}' property!`);
 
-		case 'lower-triangular':
-		if(checkSquare(arr)){
-		for(let i = 0, j = i + 1; i < arr.length && j < arr[0].length; i++, j++){
-				if(arr[i][j] !== 0){
-					return false;
+			case 'lower-triangular':
+				if (checkSquare(arr)) {
+					for (let i = 0, j = i + 1; i < arr.length && j < arr[0].length; i++ , j++) {
+						if (arr[i][j] !== 0) {
+							return false;
+						}
 					}
+					return true;
 				}
-				return true;
-			}
-		throw new TypeError(`Invalid argument received: '${arr}', the matrix has to be square to check '${property}' property!`);
+				throw new TypeError(`Invalid argument received: '${arr}', the matrix has to be square to check '${property}' property!`);
 
-		default:
-			throw new TypeError(`Invalid argument '${property}' received!`);
+			default:
+				throw new TypeError(`Invalid argument '${property}' received!`);
 		}
 	}
 
-	pow(a){
+	pow(a) {
 		integerValidate(a, 'pow');
 		return matrixExpo(this.args, a);
 	}
+	transpose() {
+		let arr = this.args;
+		var col = arr.length || 0;
+		var row = arr[0].length;
+		var i, j, tarr = [];
+		for (i = 0; i < row; i++) {
```

Comment:
for iterating in `for` loop, use let in initialisation e.g.
`for (let i = 0; i < row; i++) { `